wheelSeparation: 0.160
wheelDiameter: 0.66

